src/main/java/emt/gthandler/common/tileentities/machines/multi/DE_Fusion_Crafter.java

```diff
@@ -173,12 +174,12 @@ public GT_Recipe.GT_Recipe_Map getRecipeMap() {
     @Override
     public boolean checkRecipe(ItemStack aStack) {
         ItemStack[] tInputs = getCompactedInputs();
-        //FluidStack[] tFluids = getCompactedFluids(); unused for now
+        FluidStack[] tFluids = getCompactedFluids();
 
         if (tInputs.length > 0) {
             long tVoltage = getMaxInputVoltage();
             byte tTier = (byte) Math.max(1, GT_Utility.getTier(tVoltage));
-            GT_Recipe tRecipe = EMT_RecipeAdder.sFusionCraftingRecipes.findRecipe(getBaseMetaTileEntity(), false, gregtech.api.enums.GT_Values.V[tTier], null, tInputs);
+            GT_Recipe tRecipe = EMT_RecipeAdder.sFusionCraftingRecipes.findRecipe(getBaseMetaTileEntity(), false, gregtech.api.enums.GT_Values.V[tTier], tFluids, tInputs);
```

Comment:
It's a bit cleaner to just import V and use it than this but up to you. Not really important.

Reply:
Done